modules/apps/document-library/document-library-video/src/main/resources/META-INF/resources/preview.jsp

```diff
@@ -23,6 +23,6 @@
 		FileVersion fileVersion = (FileVersion)request.getAttribute(FileVersion.class.getName());
 		%>
 
-		<%= dlVideoRenderer.renderHTML(fileVersion, request) %>
+		<%= (dlVideoRenderer != null) ? dlVideoRenderer.renderHTML(fileVersion, request) : null %>
```

Comment:
same here. if is null maybe is an error and this can prevent it to show

Reply:
I will remove this change, as it makes sense an error should appear if dlVideoRenderer is null here.